manager/controlapi/service.go

```diff
@@ -141,6 +141,18 @@ func validateContainerSpec(container *api.ContainerSpec) error {
 	return nil
 }
 
+func validatePluginSpec(plugin *api.PluginSpec) error {
+	if plugin.Image == "" {
+		return grpc.Errorf(codes.InvalidArgument, "PluginSpec: image reference must be provided")
+	}
+
+	if _, err := reference.ParseNormalizedNamed(plugin.Image); err != nil {
```

Comment:
Tags are mandatory for plugins. If you don't specify one, it defaults to "latest". But I'm pretty sure you can't use only a digest.